### PR TITLE
[16.0][ADD] hr_department_analytic: init

### PR DESCRIPTION
**Pull request by AnizR**
This new module allows to specify an analytic account on hr department

### CODE REVIEW

File: hr_department_analytic/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Hr Department Analytic Temp",
```
Comment (adrienpeiffer) [inline hr_department_analytic/__manifest__.py]: Remove Temp

File: hr_department_analytic/README.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
+===========================
+Hr Department Analytic Temp
```
Comment (adrienpeiffer) [inline hr_department_analytic/README.rst]: Remove Temp

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

**Review by adrienpeiffer — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**